test/controllers/friendships_controller_test.rb

```diff
@@ -0,0 +1,54 @@
+require 'test_helper'
```

Comment:
Add magic comment.

Reply:
what is magic comment ma'am?

Reply:
I am talking about the frozen literal string comment.

Please go through the magic comments that rails provide.

Reply:
ok ma'am